hapi/src/config/index.js

```diff
@@ -2,19 +2,19 @@ const massive = require('massive')
 
 const massiveConfig = require('./server.config')
 const massiveDB = (async () => {
-    try {
-        return await massive(massiveConfig)
-    } catch (err) {
-        console.log('---- Failed to connect to database ----')
-        console.log(err)
+  try {
+    return await massive(massiveConfig)
+  } catch (err) {
+    console.log('---- Failed to connect to database ----')
+    console.log(err)
 
-        return null
-    }
+    return null
+  }
 })()
 
 module.exports = {
-    massiveConfig,
-    massiveDB,
-    generalContractScope: process.env.HAPI_RATING_CONTRACT || 'rateproducer',
-    edenContractScope: 'eden'
-}
\ No newline at end of file
+  massiveConfig,
+  massiveDB,
+  generalContractScope: process.env.HAPI_RATING_CONTRACT || 'rateproducer',
+  edenContractScope: 'eden'
```

Comment:
@AngeloCG97 there is a global variable for this, it is `edenContractScope`, try to use it